go.mod

```diff
@@ -5,94 +5,111 @@ go 1.23
 toolchain go1.23.1
 
 require (
-	github.com/bufbuild/buf v1.30.0
+	github.com/bufbuild/buf v1.41.0
 	github.com/crossplane/crossplane-runtime v1.17.0
 	github.com/go-json-experiment/json v0.0.0-20240815175050-ebd3a8989ca1
 	github.com/go-logr/logr v1.4.2
 	github.com/go-logr/zapr v1.3.0
 	github.com/google/go-cmp v0.6.0
 	github.com/pkg/errors v0.9.1
-	github.com/upbound/provider-aws v0.47.4
+	github.com/upbound/provider-aws v1.13.1
 	go.uber.org/zap v1.27.0
 	google.golang.org/grpc v1.66.2
 	google.golang.org/grpc/cmd/protoc-gen-go-grpc v1.5.1
-	google.golang.org/protobuf v1.34.2
+	google.golang.org/protobuf v1.34.3-0.20240816073751-94ecbc261689
 	k8s.io/api v0.30.0
 	k8s.io/apimachinery v0.30.0
 	k8s.io/utils v0.0.0-20240902221715-702e33fdd3c3
 	sigs.k8s.io/yaml v1.4.0
 )
 
 require (
-	buf.build/gen/go/bufbuild/protovalidate/protocolbuffers/go v1.33.0-20240221180331-f05a6f4403ce.1 // indirect
-	connectrpc.com/connect v1.15.0 // indirect
-	connectrpc.com/otelconnect v0.7.0 // indirect
+	buf.build/gen/go/bufbuild/bufplugin/protocolbuffers/go v1.34.2-20240904181154-a0be11449112.2 // indirect
```

Comment:
Nit: We should probably do the same we are doing for Crossplane snd just run it without importing it

Reply:
The issue is not whether it works, but the fact that they pull in a bunch of dependency that often conflict with stuff we care about, like Kubernetes dependencies